src/v2/view-builder/views/IdentifierView.js

```diff
@@ -148,6 +148,12 @@ const Body = BaseForm.extend({
       return newSchema;
     });
 
+    // If showOptionForKeepMeSignedIn is explicitly set to false, we ensure we don't display
+    // the "Keep me signed in" checkbox
+    if (this.settings.get('features.showOptionForKeepMeSignedIn') === false) {
```

Comment:
Can you simplify the name to `showKeepMeSignedIn` ? `showOption...` makes me think there's a checkbox I can check to configure this setting.

Reply:
Shouldn't this be added to the Settings class?